curvefs/src/client/rpcclient/task_excutor.cpp

```diff
@@ -72,6 +79,41 @@ int TaskExecutor::DoRPCTask(std::shared_ptr<TaskContext> task) {
     return retCode;
 }
 
+void TaskExecutor::DoAsyncRPCTask(TaskExecutorDone *done) {
```

Comment:
above synchronous processing can be implemented by this asynchronous, so the main logic here doesn't have to repeat twice.

Reply:
yeap. sync and async can be solved in same DoRPCTask distinguished by whether done is null.

Reply:
no,  the synchronous will retry in this function, but asynchronous way retry by closure return, they not same